client/state/memberships/subscriptions/actions.js

```diff
@@ -21,11 +21,15 @@ export const requestSubscriptionStop = ( subscriptionId ) => {
 		} );
 		return wpcom.req
 			.post( `/me/memberships/subscriptions/${ subscriptionId }/cancel` )
-			.then( () => {
-				dispatch( {
-					type: MEMBERSHIPS_SUBSCRIPTION_STOP_SUCCESS,
-					subscriptionId,
-				} );
+			.then( ( response ) => {
+				if ( response.redirect ) {
+					window.location = response.redirect;
```

Comment:
Can you add a comment to explain where the redirection will lead, so it is easier to understand?